File: realtime202301/src/main/java/com/gec/realtime/domain/CourseClickCount.scala
package com.gec.realtime.domain

/**
 * 实战课程点击数实体类
 * @param day_course  对应的就是HBase中的rowkey 20230614_128
 * @param click_count 对应的访问总数 42
 */

case class CourseClickCount(day_course:String, click_count:Long)

File: realtime_echats/src/main/java/com/gec/realtime/entity/CourseClickCount.java
package com.gec.realtime.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName(value="course_clickcount")
public class CourseClickCount {
//    create table course_clickcount(
//            day_course varchar(50),
//    click_count Long
//);
    private String dayCourse;
    private Long clickCount;
}